packages/react-native-renderer/src/ReactFabricEventEmitter.js

```diff
@@ -19,11 +19,11 @@ import type {TopLevelType} from 'events/TopLevelEventTypes';
 export {getListener, registrationNameModules as registrationNames};
 
 export function dispatchEvent(
-  target: Object,
+  target: null | Object,
```

Comment:
In what cases is `target` not a Fiber? Like could this type be `null | Fiber`?

Reply:
It's never not a Fiber, hence the case below, but the caller cannot guarantee it right now.

Reply:
Ultimately it is because the "internalInstanceHandle" is supposed to be an opaque object as far as the host configs are concerned. So it should really be using opaque types instead.

https://github.com/facebook/react/blob/aeda7b745d9c080150704feb20ea576238a1b9a1/packages/react-native-renderer/src/ReactFabricHostConfig.js#L168

Reply:
Cool. This makes sense.